api/pkg/apis/v1alpha1/managers/stage/stage-manager.go

```diff
@@ -496,7 +495,7 @@ func (s *StageManager) HandleTriggerEvent(ctx context.Context, campaign model.Ca
 
 		for k, v := range inputs {
 			var val interface{}
-			val, err = s.traceValue(ctx, v, triggerData.Namespace, inputs, triggerData.Outputs)
+			val, err = s.traceValue(ctx, v, triggerData.Namespace, inputs, triggers, triggerData.Outputs)
```

Comment:
You assign triggerData.Inputs to context at line 437 and in traceValue twice. Could you check if both of them are required or not?

Reply:
For the one in ResumeStage, I changed the `eCtx.Trigger = status.Inputs` because status is the activation here.